app/lib/common/providers/notifiers/chat_notifiers.dart

```diff
@@ -15,7 +15,7 @@ class AsyncConvoNotifier extends FamilyAsyncNotifier<Convo?, String> {
   late StreamSubscription<bool> _poller;
 
   @override
-  FutureOr<Convo> build(String arg) async {
+  FutureOr<Convo?> build(String arg) async {
```

Comment:
is already the existing type but by allowing `null` here, we can easily mock this provider.